src/schedule/zephyr_dma_domain.c

```diff
@@ -24,6 +24,8 @@
 
 LOG_MODULE_DECLARE(ll_schedule, CONFIG_SOF_LOG_LEVEL);
 
+#define ZEPHYR_DMA_DOMAIN_PRIO 5
```

Comment:
Is there any central place where this define might be more appropriate? So as to be able to decide on ordering some things.

I like the idea behind the patch.

Reply:
@LaurentiuM1234 how did you decide on value 5?
How to you know the priority is less than logging?

Reply:
Could this be based on a Zephyr Kconfig around the logging thread priorty ? e.g. have a Kconfig to make the Zephyr logging preemptive so that DMA pipeline preempts logging ?

Reply:
You could do that if you know that it always needs to be "one greater than logging" for example. Otherwise the way I might go about it is exposing both to Kconfig and in the build Kconfig .prj file putting all the prio's nicely next to each other with some comments around it.

A BUILD_ASSERT() can be done with this as well to ensure that DMA_PRIO > LOG_PRIO at build time if you like.